### PR TITLE
Use setuptools scm

### PR DESCRIPTION
**Pull request by ocefpaf**
Review and merge after #69

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+release:
+	rm -rf dist/
+	python setup.py sdist bdist_wheel
+	twine upload dist/*
+
+isort:
+	isort --remove-import . --apply --use-parentheses --trailing-comma
+
+format:
+	black --line-length 79 .
```
Comment (Conengmo) [inline Makefile:10]: I personally see no reason to not use the default line length of black here.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+release:
+	rm -rf dist/
+	python setup.py sdist bdist_wheel
+	twine upload dist/*
+
+isort:
+	isort --remove-import . --apply --use-parentheses --trailing-comma
```
Comment (Conengmo) [inline Makefile:7]: these options are not necessary because the settings in `setup.cfg`?

Comment (ocefpaf) [reply Makefile:7]: Actually. I have a new way of implementing this with pre-commit hooks, and GH actions checks for those who don't have pre-commits. Let me update this PR and, if you like that, we can implement it on `folium` too.

Comment (Conengmo) [reply Makefile:7]: Alright, sounds interesting!

**Comment by ocefpaf** [comment]
Should probably resend this one.

**Pull request closed without merging**